### PR TITLE
starting stats pages

### PR DESCRIPTION
**Pull request by aspina7**
Just getting started on basic stats pages - still WIP

### CODE REVIEW

File: pages/descriptive_statistics.Rmd [hunk 1]
```diff
@@ -22,150 +22,427 @@ knit: (function(inputFile, encoding) {
...
+```
+
+### `skimr` package
+
+Using the `skimr` package you can get a more detailed overview of each of the variables in your dataset.
+
+```{r descriptive_browse_skimr}
+
+## get information about each variable in a dataset 
+skim(linelist)
```
Comment (nsbatra) [inline pages/descriptive_statistics.Rmd:145]: Nice I didn't know this one!

**Review by nsbatra — APPROVED** [review]
Thanks Alex! Looking forward to seeing more

File: pages/descriptive_statistics.Rmd [hunk 2]
```diff
@@ -22,150 +22,745 @@ knit: (function(inputFile, encoding) {
...
+  tbl_summary(
+    ## specify the grouping variable
+    by = outcome
+  ) %>% 
+  ## specify what test want to perform
+  add_p()
+
+```
+
+### `dplyr` package
```
Comment (aspina7) [inline pages/descriptive_statistics.Rmd:595]: there must be less dense ways to do this that i am missing?!

File: pages/descriptive_statistics.Rmd [hunk 3]
```diff
@@ -22,150 +22,745 @@ knit: (function(inputFile, encoding) {
...
+  filter(!is.na(gender)) %>% 
+  mutate(pval = chisq.test(linelist$gender, linelist$outcome)$p.value)
+```
 
 <!-- ======================================================= -->
-## Resources {.tabset .tabset-fade .tabset-pills}
 
-This tab should stay with the name "Resources".
-Links to other online tutorials or resources.
+## Correlations {.tabset .tabset-fade .tabset-pills}
```
Comment (aspina7) [inline pages/descriptive_statistics.Rmd:740]: need a better dataset to demonstrate correlations - ideally with numeric variables that are associated. 
If nothing then maybe just best to use the examples from the corrr website (mtcars, airquality or penguins....)

Comment (nsbatra) [reply pages/descriptive_statistics.Rmd:740]: Would a few extra numeric columns in linelist be sufficient? (e.g. temperature, CT value, etc.)? Or do we need a whole other dataset?

**Review by nsbatra — APPROVED** [review]
Approving now so you can do what you wish. I can come back with other comments later when I have time.

File: pages/descriptive_statistics.Rmd (hunk 3 above)
Comment (aspina7) [reply pages/descriptive_statistics.Rmd:740]: yeah guess we could check in weight height temp and ct or something?

**Comment by nsbatra** [comment]
Ok I will work to add those columns today or tomorrow.

File: pages/stat_tests.Rmd [hunk 4]
```diff
@@ -22,72 +22,299 @@ knit: (function(inputFile, encoding) {
...
+  ## merge with the outputs of the regression 
+  bind_cols(., model) %>% 
+  ## only keep columns interested in 
+  select(term, 2:3, estimate, conf.low, conf.high, p.value)
+
+
+```
+
+
+To run over several exposure variables to produce univariate odds ratios (i.e.
```
Comment (aspina7) [inline pages/stat_tests.Rmd:201]: not sure if all of this purrr::map stuff is way too dense?

**Review by nsbatra — APPROVED** [review]

File: pages/descriptive_statistics.Rmd [hunk 5]
```diff
@@ -22,150 +22,746 @@ knit: (function(inputFile, encoding) {
...
+```
+
+#### Kruskal-wallis test
+
+Compare the distribution of a continuous variable in two or more groups, 
+regardless of whether the data is normally distributed. 
+
+```{r kruskal_gt}
+
+linelist %>%
```
Comment (nsbatra) [inline pages/descriptive_statistics.Rmd:561]: @aspina7 as we discussed I'd suggest adding a tab covering the use of one-liner functions like t.test(), wilcox.test(), chisq.test(), cor(), etc... These outputs will be very attractive to many people, but it probably best to offer an alt

**Review by nsbatra — DISMISSED** [review]
Thanks for adding this!

**Review by nsbatra — APPROVED** [review]

**Comment by aspina7** [comment]
merging for now - will start a new branch to deal with stratified reg stuff later

**Pull request merged**